packages/datadog-plugin-prisma/src/engine.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+const { CLIENT_PORT_KEY } = require('../../dd-trace/src/constants')
+
+const databaseDriverMapper = {
+  postgresql: {
+    type: 'sql',
+    'db.type': 'postgres'
+  },
+  mysql: {
+    type: 'sql',
+    'db.type': 'mysql'
+  },
+  mongodb: {
+    type: 'mongodb',
+    'db.type': 'mongodb'
+  },
+  sqlite: {
+    type: 'sql',
+    'db.type': 'sqlite'
+  }
+}
+
+class PrismaEngine extends DatabasePlugin {
+  static get id () { return 'prisma' }
+  static get operation () { return 'engine' }
+  static get system () { return 'prisma' }
+
+  start (ctx) {
+    // console.log('PrismaEngine start called', ctx.engineSpan)
+    const { engineSpan, allEngineSpans, childOf, dbConfig } = ctx
+
+    const service = this.serviceName({ pluginConfig: this.config, system: this.system })
+
+    const unixStartTime = hrTimeToUnixTimeMs(engineSpan.startTime)
+
+    const spanName = engineSpan.name.substring(14) // remove 'prisma:engine:' prefix
```

Comment:
```suggestion
    const spanName = engineSpan.name.slice(14) // remove 'prisma:engine:' prefix
```

Substring has some behavior that is more difficult to follow in JS

Reply:
👍